apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -219,6 +219,8 @@ class AppPreferences {
   }, settingsThemes)
   val themeOverrides = mkThemeOverridesPreference()
   val profileImageCornerRadius = mkFloatPreference(SHARED_PREFS_PROFILE_IMAGE_CORNER_RADIUS, 22.5f)
+  val chatItemRoundness = mkFloatPreference(SHARED_PREFS_CHAT_ITEM_ROUNDNESS, 0.75f)
```

Comment:
take a look at `SHARED_PREFS_PROFILE_IMAGE_CORNER_RADIUS` and use similar name and idea (number in range 0 .. 100, not 0 .. 1)
So `SHARED_PREFS_CHAT_ITEM_CORNER_RADIUS` is ok. If something like this already used in iOS, it should be changed there too to apply the same format and naming.

Reply:
Discussed offline